### PR TITLE
feat(org-stats): Final UI tweaks

### PR DESCRIPTION
**Pull request by leedongwei**
### Before 
Screen Shot 2021-04-27 at 1 09 02 PM

### After 
Screen Shot 2021-04-27 at 1 10 49 PM

### Overall Page
Screen Shot 2021-04-27 at 1 59 33 PM

### CODE REVIEW

**Comment by leedongwei** [comment]
@imatwawana FYI

**Comment by dashed** [comment]
Are you sure this is final? 😸

**Comment by leedongwei** [comment]
@dashed final_final_v2

**Comment by imatwawana** [comment]
> @imatwawana FYI

LGTM!

File: static/app/views/usageStats/usageStatsOrg.tsx [hunk 1]
```diff
@@ -253,22 +253,27 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
         }
 
         count.total += group.totals['sum(quantity)'];
-        count[outcome] += group.totals['sum(quantity)'];
+
+        if (outcome in count) {
+          count[outcome] += group.totals['sum(quantity)'];
+        } else {
```
Comment (JoshFerge) [inline static/app/views/usageStats/usageStatsOrg.tsx:259]: i don't understand this, aren't all the outcomes in count?

File: static/app/views/usageStats/usageChart/index.tsx [hunk 2]
```diff
@@ -199,17 +205,23 @@ export class UsageChart extends React.Component<Props, State> {
...
 
   get chartColors() {
-    const {dataCategory} = this.props;
+    const {dataCategory, chartHighlightAccepted} = this.props;
 
     if (dataCategory === DataCategory.ERRORS) {
-      return [COLOR_ERRORS, COLOR_ERRORS_DROPPED, COLOR_PROJECTED];
+      return chartHighlightAccepted
+        ? [COLOR_ERRORS, COLOR_ERRORS_LIGHT, COLOR_PROJECTED]
+        : [COLOR_ERRORS_LIGHT, COLOR_ERRORS, COLOR_PROJECTED];
```
Comment (maheskett) [inline static/app/views/usageStats/usageChart/index.tsx:213]: is this prop needed? I think it'd be nice for all usage charts to use the same colors per outcome

File: static/app/views/usageStats/usageStatsOrg.tsx (hunk 1 above)
Comment (JoshFerge) [reply static/app/views/usageStats/usageStatsOrg.tsx:259]: (I believe the abuse outcome is unused in sentry and are recorded with an org_id of 0 so they would never show up in our queries) https://github.com/getsentry/ops/blob/999bae6f4f34d6716f50158680122e937a82d815/cookbooks/td-agent/recipes/nginx.rb

File: static/app/views/usageStats/usageChart/index.tsx (hunk 2 above)
Comment (leedongwei) [reply static/app/views/usageStats/usageChart/index.tsx:213]: For the subscription page, it's better to have an emphasis on `accepted`, so it's easy to see usage against quota.

For the org stats page, product wants the page to aid sales folks, so highlighting `dropped` is more useful.

File: static/app/views/usageStats/usageStatsOrg.tsx (hunk 1 above)
Comment (leedongwei) [reply static/app/views/usageStats/usageStatsOrg.tsx:259]: Just wanted to make sure that any unexpected outcome would fall into `dropped`.

**Review by JoshFerge — APPROVED** [review]
code looks good to me. will let others chime in on colors etc.

**Review by maheskett — APPROVED** [review]

**Review by doralchan — APPROVED** [review]
Dementors @leedongwei

**Pull request merged**